idaes_examples/notebooks/docs/unit_models/custom_unit_models/creating_unit_model.ipynb

```diff
@@ -0,0 +1,2813 @@
+    "        ) == pytest.approx(0.000403124, rel=1e-5)\n",
+    "        assert value(\n",
+    "            model.fs.unit.aqueous_outlet.conc_mass_comp[0, \"CaSO4\"]\n",
+    "        ) == pytest.approx(0.0985, rel=1e-5)\n",
+    "        assert value(\n",
+    "            model.fs.unit.aqueous_outlet.conc_mass_comp[0, \"KNO3\"]\n",
+    "        ) == pytest.approx(0.194, rel=1e-5)\n",
+    "        assert value(\n",
+    "            model.fs.unit.aqueous_outlet.conc_mass_comp[0, \"NaCl\"]\n",
+    "        ) == pytest.approx(0.146775, rel=1e-5)\n",
+    "\n",
+    "        # Checking for outlet temperature\n",
+    "        assert value(model.fs.unit.organic_outlet.temperature[0]) == pytest.approx(\n",
+    "            300, rel=1e-5\n",
+    "        )\n",
+    "        assert value(model.fs.unit.aqueous_outlet.temperature[0]) == pytest.approx(\n",
+    "            300, rel=1e-5\n",
+    "        )\n",
+    "\n",
+    "        # Checking for outlet pressure\n",
+    "        assert value(model.fs.unit.organic_outlet.pressure[0]) == pytest.approx(\n",
+    "            1, rel=1e-5\n",
+    "        )\n",
+    "        assert value(model.fs.unit.aqueous_outlet.pressure[0]) == pytest.approx(\n",
+    "            1, rel=1e-5\n",
+    "        )\n",
+    "\n",
+    "        # Fixed state variables\n",
+    "        assert model.fs.unit.organic_inlet.flow_vol[0].fixed\n",
+    "        assert model.fs.unit.organic_inlet.conc_mass_comp[0, \"NaCl\"].fixed\n",
+    "        assert model.fs.unit.organic_inlet.conc_mass_comp[0, \"KNO3\"].fixed\n",
+    "        assert model.fs.unit.organic_inlet.conc_mass_comp[0, \"CaSO4\"].fixed\n",
+    "        assert model.fs.unit.organic_inlet.temperature[0].fixed\n",
+    "        assert model.fs.unit.organic_inlet.pressure[0].fixed\n",
+    "\n",
+    "        assert model.fs.unit.aqueous_inlet.flow_vol[0].fixed\n",
+    "        assert model.fs.unit.aqueous_inlet.conc_mass_comp[0, \"NaCl\"].fixed\n",
+    "        assert model.fs.unit.aqueous_inlet.conc_mass_comp[0, \"KNO3\"].fixed\n",
+    "        assert model.fs.unit.aqueous_inlet.conc_mass_comp[0, \"CaSO4\"].fixed\n",
+    "        assert model.fs.unit.aqueous_inlet.temperature[0].fixed\n",
+    "        assert model.fs.unit.aqueous_inlet.pressure[0].fixed\n",
+    "\n",
+    "    @pytest.mark.component\n",
+    "    def test_structural_issues(self, model):\n",
+    "        dt = DiagnosticsToolbox(model)\n",
+    "        dt.assert_no_structural_warnings()"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "idaes-pse",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.13"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
Add a summary at the end to highlight the main takeaways/goals/lessons from this notebook.